### DISCUSSION TITLE
카카오 백신예약 사용법을 보고도 모르겠습니다 ㅠㅠ

### DISCUSSION THREAD

**Discussion by com-is-not-easy** [changdoc/naver-vaccine-macro#181] [Q&A]
제가 이 방법을 지인소개로 알게되었는데 명시된 순서대로 그대로 하면되고 쉽다고 해서 보니 제가 못하는 건지 아님 제 컴퓨터 어디 설정에서 막혀서 진도가 안나가는 건지 너무 어렵네요 ㅠㅠ 제 생각엔 둘다 인것같은데 도와주세요 ㅠㅠ

문제점 1
![image](https://user-images.githubusercontent.com/88299588/127800336-046954fb-f366-443e-a1db-5af5f13d1e02.png)
> 여기서 하단 새로고침 누르면 post 가 뜬다고 했는데 아무것도 안떠요
> 만약 여기서 뭐가 뜬가고 하면 그게 좌표?라고 하시는데 그거 뜬다음에 그대로 둬요? 아님 뭔갈 더 해야하나요? 

문제점 2
![image](https://user-images.githubusercontent.com/88299588/127800477-bc0e302f-8966-4c0c-9fd4-589c84764c96.png)
> 제 컴퓨터가 윈도우라 저기 4개중에 윈도우만 다운받아서 실행시켜보니 검정색 창이 뜨면서 
![image](https://user-images.githubusercontent.com/88299588/127800570-2b7c15c8-1777-4573-9958-d257aa92094f.png)
이렇게 나옵니다...;;; 분명 카카오로그인유지 다 한거 다 체크하고 이거 실행시켰는데 이래요

마음같아서는 컴퓨터 화면공유하기 해가지고 직접 보여드리고 싶은 마음이 굴뚝같을 정도로 진도가 너무 안나가고 뭐가 원인인지도 모르겠고 뭘 손대야하는지도 모르겠습니다 ㅠㅠㅠㅠ

**Comment by changdoc** [comment]
문제점 1은 첨부해주신 스크린샷은 정상동작하는것 처럼 보이는데요. 잔여 백신 발생시 네이버 예약 신청 페이지로 잘 이동 되는지만 체크 하시면 될 것 같습니다.

문제점 2는 제가 만든 매크로가 아니라서 답변을 드릴 수가 없을것 같습니다.

**Reply by com-is-not-easy** [reply]
우선 제 글 읽어주셔서 너무 감사합니다 ㅠㅠ 
사용법으로 글을 올리는 신 것에 보면
1.Chrome 브라우저를 이용하여 모든 작업을 진행하므로, 크롬 브라우저를 실행해주세요.
2.카카오 계정 로그인 페이지 에 접근하여 카카오 로그인을 합니다. 로그인 시 주의사항
3.#2를 보고 잔여백신을 검색할 범위의 좌표값을 찾습니다.
4.Release 페이지에서 본인의 운영체제에 맞는 파일을 다운로드 받고 실행합니다.
5.사용자 권한 동의를 요청하면 승인해주세요. 자동으로 검색 및 예약시도를 진행합니다.
6.예약 성공 시 빵빠레 소리와 함께 예약이 성공했음이 안내됩니다.

이렇게 적혀있잔아요?
2번까지는 완벽하게 하였고 3번 좌표값찾는다고 f12누르고 들어가서 띄운게 문제점1의 스샷이에요 그런데 그다음에 새로고침누르면  post가 떠야하는데 안떠서 멈춘상태에서 뭔가 설치가 안되있어서 그런가 싶어서 건너뛰고 4번에 릴리스페이지들어가서 뭐 파일일 있길래 설치했는데 그 마저도 안되는게 마지막 검은화면 이에요 선생님 ㅠㅠㅠㅠ
뭔가 잘못됬는데 뭐가 잘못된지 모르겠습니다 ...

**Reply by changdoc** [reply]
음.. 여쭤보신 부분은 제가 만든 매크로가 아니라 https://github.com/SJang1/korea-covid-19-remaining-vaccine-macro/discussions
이곳에 질문을 하셔야할 것 같습니다.

**Reply by com-is-not-easy** [reply]
핫 감사합니다!!
